servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -101,6 +101,26 @@ enum RemovalReason {
   TOPIC_DIVERSITY
 }
 
+"""
+Reasons for removing SectionItems from ML Sections.
+
+This is used by ML downstream to improve their modeling.
+"""
+enum SectionItemRemovalReason {
```

Comment:
I don't think we need to expose `deactivatedReasons` on `SectionItem` type. This will be used by ML & (for now) accessed from DB directly.